app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationController < ActionController::Base
+end
```

Comment:
- [ ] I will encourage you to add the two methods, the index and show, even if they will be empty. Remember to add it for user conteroller too.